app/javascript/dashboard/components-next/Inbox/InboxCard.vue

```diff
@@ -0,0 +1,235 @@
+import { getInboxIconByType } from 'dashboard/helper/inbox';
+import { dynamicTime, shortTimestamp } from 'shared/helpers/timeHelper';
+import {
+  snoozedReopenTimeToTimestamp,
+  shortenSnoozeTime,
+} from 'dashboard/helper/snoozeHelpers';
+import { NOTIFICATION_TYPES_MAPPING } from 'dashboard/routes/dashboard/inbox/helpers/InboxViewHelpers';
+
+import Icon from 'dashboard/components-next/icon/Icon.vue';
+import Avatar from 'dashboard/components-next/avatar/Avatar.vue';
+import CardPriorityIcon from 'dashboard/components-next/Conversation/ConversationCard/CardPriorityIcon.vue';
+import SLACardLabel from 'dashboard/components-next/Conversation/ConversationCard/SLACardLabel.vue';
+import InboxContextMenu from 'dashboard/routes/dashboard/inbox/components/InboxContextMenu.vue';
+
+const props = defineProps({
+  inboxItem: { type: Object, default: () => ({}) },
+  stateInbox: { type: Object, default: () => ({}) },
+});
+
+const emit = defineEmits([
+  'click',
+  'contextMenuOpen',
+  'contextMenuClose',
+  'markNotificationAsRead',
+  'markNotificationAsUnRead',
+  'deleteNotification',
+]);
+
+const { t } = useI18n();
+
+const isContextMenuOpen = ref(false);
+const contextMenuPosition = ref({ x: null, y: null });
+const slaCardLabel = ref(null);
+
+const getMessageClasses = {
+  emphasis: 'text-sm font-medium text-n-slate-11',
+  emphasisUnread: 'text-sm font-medium text-n-slate-12',
+  normal: 'text-sm font-normal text-n-slate-11',
+  normalUnread: 'text-sm text-n-slate-12',
+};
+
+const primaryActor = computed(() => props.inboxItem?.primaryActor);
+const meta = computed(() => primaryActor.value?.meta);
+const assigneeMeta = computed(() => meta.value?.sender);
+const isUnread = computed(() => !props.inboxItem?.readAt);
+const inbox = computed(() => props.stateInbox);
+
+const inboxIcon = computed(() => {
+  const { phoneNumber, channelType } = inbox.value;
+  return getInboxIconByType(channelType, phoneNumber);
+});
+
+const hasSlaThreshold = computed(() => {
+  return slaCardLabel.value?.hasSlaThreshold && primaryActor.value?.slaPolicyId;
+});
+
+const lastActivityAt = computed(() => {
+  const timestamp = props.inboxItem?.lastActivityAt;
+  return timestamp ? shortTimestamp(dynamicTime(timestamp)) : '';
+});
+
+const menuItems = computed(() => [
+  { key: 'delete', label: t('INBOX.MENU_ITEM.DELETE') },
+  {
+    key: isUnread.value ? 'mark_as_read' : 'mark_as_unread',
+    label: t(`INBOX.MENU_ITEM.MARK_AS_${isUnread.value ? 'READ' : 'UNREAD'}`),
+  },
+]);
+
+const formatPushMessage = message => {
+  return message.replace(/^([^:]+):/g, (match, name) => {
+    return `<span class="${isUnread.value ? getMessageClasses.emphasisUnread : getMessageClasses.emphasis}">${name}:</span>`;
+  });
+};
+
+const formattedMessage = computed(() => {
+  const messageContent = `<span class="${isUnread.value ? getMessageClasses.normalUnread : getMessageClasses.normal}">${formatPushMessage(props.inboxItem?.pushMessageBody || '')}</span>`;
```

Comment:
It would be better we can use computed repeated calculations for message classes.
```
const messageClasses = computed(() => ({
  emphasis: isUnread.value ? getMessageClasses.emphasisUnread : getMessageClasses.emphasis,
  normal: isUnread.value ? getMessageClasses.normalUnread : getMessageClasses.normal,
}));
```

```
const messageContent = `<span class="${messageClasses.value.normal}">${formatPushMessage(props.inboxItem?.pushMessageBody || '')
```

Reply:
@muhsin-k Done https://github.com/chatwoot/chatwoot/pull/10599/commits/d0b8fb37cce45ba9d19a68e52e253791d603818e